data/xml/2021.emnlp.xml

```diff
@@ -631,14 +631,16 @@
       <author><first>Aylin</first><last>Caliskan</last></author>
       <pages>518–532</pages>
       <abstract>We use a dataset of U.S. first names with labels based on predominant gender and racial group to examine the effect of training corpus frequency on tokenization, contextualization, similarity to initial representation, and bias in BERT, GPT-2, T5, and XLNet. We show that predominantly female and non-white names are less frequent in the training corpora of these four language models. We find that infrequent names are more self-similar across contexts, with Spearman’s rho between frequency and self-similarity as low as -.763. Infrequent names are also less similar to initial representation, with Spearman’s rho between frequency and linear centered kernel alignment (CKA) similarity to initial representation as high as .702. Moreover, we find Spearman’s rho between racial bias and name frequency in BERT of .492, indicating that lower-frequency minority group names are more associated with unpleasantness. Representations of infrequent names undergo more processing, but are more self-similar, indicating that models rely on less context-informed representations of uncommon and minority names which are overfit to a lower number of observed contexts.</abstract>
-      <url hash="df05fda7">2021.emnlp-main.41</url>
+      <url hash="a79a9df3">2021.emnlp-main.41</url>
       <attachment type="Software" hash="b79e0e4c">2021.emnlp-main.41.Software.zip</attachment>
       <bibkey>wolfe-caliskan-2021-low</bibkey>
       <doi>10.18653/v1/2021.emnlp-main.41</doi>
       <video href="2021.emnlp-main.41.mp4"/>
       <pwcdataset url="https://paperswithcode.com/dataset/bookcorpus">BookCorpus</pwcdataset>
       <pwcdataset url="https://paperswithcode.com/dataset/openwebtext">OpenWebText</pwcdataset>
       <pwcdataset url="https://paperswithcode.com/dataset/webtext">WebText</pwcdataset>
+      <revision id="1" href="2021.emnlp-main.41v1" hash="df05fda7"/>
+      <revision id="2" href="2021.emnlp-main.41v2" hash="a79a9df3" date="2022-11-30">Corrected a bug in code and updated result in the paper.</revision>
     </paper>
```

Comment:
This is not a revision, but an erratum.

Reply:
This link now returns the erratum, not the paper. https://aclanthology.org/2021.emnlp-main.41.pdf

So change this to an erratum, and then fix the PDFs. In the future you can use the `-e` flag to the `add_revision.py` script.